AcsEmulator/AcsEmulatorAPI/Models/CommunicationIdentifier.cs

```diff
@@ -46,4 +46,6 @@ public enum CommunicationIdentifierKind
     public record CommunicationUser(string Id);
 
     public record PhoneNumber(string Value);
-}
+
+    public record CommunicationIdentifierModel(string kind, CommunicationUser communicationUser);
```

Comment:
This is a dupe of the `CommunicationIdentifier`, please use the existing model instead. If it confuses you that the type is named differently from the swagger, then just rename CommunicationIdentifier to CommunicationIdentifierModel